src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -719,9 +719,16 @@ public static EntityTeleportEvent.EnderEntity onEnderTeleport(LivingEntity entit
         return event;
     }
 
+    @ApiStatus.Internal
     public static EntityTeleportEvent.EnderPearl onEnderPearlLand(ServerPlayer entity, double targetX, double targetY, double targetZ, ThrownEnderpearl pearlEntity, float attackDamage)
     {
-        EntityTeleportEvent.EnderPearl event = new EntityTeleportEvent.EnderPearl(entity, targetX, targetY, targetZ, pearlEntity, attackDamage);
+        return onEnderPearlLand(entity, targetX, targetY, targetZ, pearlEntity, attackDamage, null);
+    }
```

Comment:
This should be annotated with `@Deprecated(forRemoval = true, since = "1.19.2")`